<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileSettingsWidget</class>
 <widget class="QWidget" name="ProfileSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="deviceLabel">
       <property name="text">
        <string>Device:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="deviceComboBox">
       <item>
        <property name="text">
         <string>Chromasound Nova</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chromasound Nova Direct</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chromasound Pro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chromasound Pro Direct</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SEGA Genesis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ASCII MSX2</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
       <item>
        <widget class="QLabel" name="pcmStrategyLabel">
         <property name="text">
          <string>PCM Strategy:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="pcmStrategyComboBox">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Inline</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sequential</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Random</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="chromasoundCheckBox">
       <property name="text">
        <string>Support Chromasound VGM shorthand commands</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="discretePCMCheckBox">
       <property name="text">
        <string>Use discrete PCM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="pcmSRAMCheckBox">
       <property name="text">
        <string>Use PCM SRAM</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
